wordpress/web/app/mu-plugins/bioco-core/bioco-core.php

```
@@ -0,0 +1,82 @@
+define('BIOCO_CORE_DIR', __DIR__);
+
+require_once BIOCO_CORE_DIR . '/includes/helpers.php';
+
+/**
+ * ACF Local JSON — this plugin's own acf-json/ dir. The fleet move is complete,
+ * so only bioco-core may load field groups; theme JSON paths were a temporary
+ * migration-window state (see PORTING-THEME-SWAP.md pattern 3).
+ */
+add_filter('acf/settings/save_json', fn() => BIOCO_CORE_DIR . '/acf-json');
+add_filter('acf/settings/load_json', function ($paths) {
+    $theme_root = trailingslashit(wp_normalize_path(WP_CONTENT_DIR . '/themes'));
```

PHPStan (2.2.7)

[error] Constant WP_CONTENT_DIR not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)